components/Tabs2/src/MDCTabBarView.h

```diff
@@ -19,4 +19,13 @@
  */
 __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScrollView
 
+/** The set of items displayed in the Tab bar. */
+@property(nonnull, nonatomic, copy) NSArray<__kindof UITabBarItem *> *items;
+
+/** The currently-selected item in the Tab bar. */
+@property(nullable, nonatomic, strong) __kindof UITabBarItem *selectedItem;
+
+/** Selects one of the items with optional animation. */
+- (void)setSelectedItem:(__kindof UITabBarItem *)selectedItem animated:(BOOL)animated;
```

Comment:
I think it's better to leave the animation for a later PR and remove the `setSelectedItem:animated:` API for now. This PR can focus on the basics of item management and selection. Once we have an Example VC in-place, we can start adding animation so we can verify the timing and appearance.

Reply:
Sure! Then I moved all logic inside this function to selctedItem setter :)